src/templates/all/Makefile

```diff
@@ -0,0 +1,14 @@
+## DO NOT EDIT!
+# This file was provisioned by Terraform
+# File origin: https://github.com/Arrow-air/tf-github/tree/main/src/templates/all/Makefile
+
+DOCKER_NAME := arrow-${name}
+
+help: .help-base .help-cspell .help-markdown .help-editorconfig
```

Comment:
toml, python, rust, docker helps and includes?

Reply:
This is the 'default' Makefile, it's the minimal that will be deployed to all repo's.
Overrides will be there if more tests are needed (see the Makefile.tftpl specific for rust).